test/backend/fused_op.in.cpp

```diff
@@ -347,6 +347,57 @@ NGRAPH_TEST(${BACKEND_NAME}, group_conv)
     EXPECT_EQ(expected, read_vector<float>(result0));
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, batch_mat_mul_transpose)
```

Comment:
These tests need to be guarded by #ifndef NGRAPH_JSON_DISABLE

Reply:
Did you mean to add the guards for tests in core_fusion.cpp? I will add there as they use json files as input
I don't think we need it here.

Reply:
done